rollup-common.js

```diff
@@ -379,6 +379,9 @@ const litMonoBundleConfig = ({
         'const ENABLE_EXTRA_SECURITY_HOOKS = false',
       'const ENABLE_SHADYDOM_NOPATCH = true':
         'const ENABLE_SHADYDOM_NOPATCH = false',
+      // For downleveled ES5 build of polyfill-support
+      'var ENABLE_SHADYDOM_NOPATCH = true':
+        'const ENABLE_SHADYDOM_NOPATCH = false',
```

Comment:
`const` is not ES5. Is this an issue?

Reply:
It wasn't since it gets DCE'ed and I had FUD about terser not DCE'ing with vars, but confirmed replacing to a var is fine, so I've updated to that.